usaspending_api/awards/v2/views/transactions.py

```diff
@@ -85,6 +95,8 @@ def _format_results(self, rows):
             result = {k: row[v] for k, v in self.transaction_lookup.items() if k != "award_id"}
             if result["is_fpds"]:
                 unique_prefix = "CONT_TX"
+            else:
+                result["cfda_number"] = row["cfda_number"]
```

Comment:
Not worth blocking this PR on, but if you do another round of changes at all might be good to move `unique_prefix = "ASST_TX"` into this else to keep all FABS specific parts together.